docs/en/rules/Azure.FrontDoor.Logs.md

```diff
@@ -25,6 +25,103 @@ Management operations for Front Door is captured automatically within Azure Acti
 
 Consider configuring diagnostics setting to log network activity through Front Door.
 
+## EXAMPLES
+
+<!-- 
+### Configure with Azure template
+
+To deploy Front Door that pass this rule:
+
+- Deploy a diagnostic settings sub-resource.
+- Enable logging for the `AuditEvent` category.
+
+For example:
+
+```json
+{
+    "coming soon once Bicep compiles and deploys...",
+
+}
+```
+-->
+
+---
+
+### Configure Manually
+
+To deploy Front Door that passes this rule:
+
+- Deploy a diagnostic settings sub-resource.
+- Manually go into the diagnostic settings and add these two rules:
+  - Enable logging for the `FrontdoorAccessLog` category.
+  - Enable logging for the `FrontdoorWebApplicationFirewallLog` category.
+
+---
+
+### Configure with Azure template
+
+If this setting was available in Bicep, it could be set like the following Bicep shows. 
+
+#### Bicep Template
+
+```bicep
+param frontDoorName string = 'myFrontDoor'
+param workSpaceId string = ''
+param location string = resourceGroup().location
+
+resource frontDoorResource 'Microsoft.Cdn/profiles@2021-06-01' = {
+  name: frontDoorName
+  location: 'Global'
+  sku: {
+    name: 'Standard_AzureFrontDoor'
+  }
+}
+
+resource frontDoorInsightsResource 'Microsoft.Cdn/profiles/providers/diagnosticSettings@2020-01-01-preview' = {
+  name: '${frontDoorName}/Microsoft.Insights/service'
+  dependsOn: [
+    frontDoorResource
+  ]
```

Comment:
Try these updates.

```suggestion
  scope: frontDoorResource
  name: 'service'
```

Reply:
That looks good, but when I actually put that into a Bicep template it won't compile. "The property "scope" is not allowed on objects of type "microsoft.insights/diagnosticSettings". Permissible properties include "dependsOn"."

Reply:
I've tried many different ways, none of which worked... 

scope (none) => Scope "resourceGroup" is not valid for this resource type. Permitted scopes: "managementGroup"

scope: managementGroup(managementGroupIdentifier) => A resource's scope must match the scope of the Bicep file for it to be deployable. You must use modules to deploy resources to a different scope

scope: frontDoorResource  => Scope "resource" is not valid for this resource type. Permitted scopes: "managementGroup"

scope: 'managementGroup' => The property "scope" is not allowed on objects of type "microsoft.insights/diagnosticSettings". Permissible properties include "dependsOn"

Can't seem to find that secret sauce to get this to compile yet...